components/common/link/Link.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ElementType, FC } from 'react';
+
+import NextLink from 'next/link';
+
+import MuiLink, { LinkProps } from '@mui/material/Link';
```

Comment:
Please, add `type` keyword. We can use  `LinkProps as MuiLinkProps`

Reply:
'+